src/common.make

```diff
@@ -2,8 +2,7 @@ HOST?=10.11.99.1
 ARCH?=arm
 CXX_BIN?=arm-linux-gnueabihf-g++
 CC_BIN?=arm-linux-gnueabihf-gcc
-CPP_FLAGS=-pthread -lpthread -fdata-sections -ffunction-sections -Wl,--gc-sections
-
+CPP_FLAGS=-pthread -lpthread --std=c++17 -fdata-sections -ffunction-sections -Wl,--gc-sections
```

Comment:
I'm using `inline static` to initialize static member variables, which requires at least C++17, but I'm happy to drop the requirement if you'd prefer.

Reply:
prefer not to use c++17 yet - trying to be as old as possible (but still need those shared_ptrs)